attest/verifier/src/status.rs

```diff
@@ -451,6 +665,26 @@ mod test {
         assert!(!verifier.verify(&data))
     }
 
+    /// Ensure a CONFIGURATION_AND_SW_HARDENING_NEEDED result with the expected
+    /// MRENCLAVE but an unexpected advisory fails.
+    #[test]
+    fn mrenclave_multi_config_sw_fail_short() {
+        let verifier = MrEnclaveVerifier {
+            mr_enclave: MrEnclave::from(&MR_ENCLAVE),
+            config_ids: vec!["INTEL-SA-00334".to_owned()],
+            sw_ids: vec!["INTEL-SA-00334".to_owned()],
+        };
+
+        let report = VerificationReport {
+            sig: Default::default(),
+            chain: vec![],
+            http_body: IAS_CONFIG_SW.trim().to_owned(),
```

Comment:
Based on the function name I'm guessing this is meant to be testing for a non excluded advisory for both config and software
```suggestion
    /// Ensure a CONFIGURATION_AND_SW_HARDENING_NEEDED result with the expected
    /// MRENCLAVE but an insufficient sw and config advisory allow-listing 
    /// fails.
    #[test]
    fn mrenclave_multi_config_sw_fail_short() {
        let verifier = MrEnclaveVerifier {
            mr_enclave: MrEnclave::from(&MR_ENCLAVE),
            config_ids: vec!["INTEL-SA-00334".to_owned()],
            sw_ids: vec!["INTEL-SA-00334".to_owned()],
        };

        let report = VerificationReport {
            sig: Default::default(),
            chain: vec![],
            http_body: IAS_CONFIG_SW_334_615.trim().to_owned(),
```